frontend/src/metabase-lib/parameters/utils/template-tags.ts

```diff
@@ -60,25 +60,21 @@ export function getTemplateTagParameters(
     .filter(
       tag =>
         tag.type != null &&
+        tag.type !== "card" &&
```

Comment:
I think the comment for this function needs to be updated (lines 51-52), at least the path where the clojure function is defined. Is the rest still relevant?

Reply:
Good catch, I've updated the comment. The rest is still relevant of course